build.gradle

```diff
@@ -30,16 +30,6 @@ dependencies {
     compile group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.14.1'
 }
 
-cucumber {
```

Comment:
this part is use to generate report, or it changed

Reply:
Both could be used, that shouldn't have been removed since it is supposed to run tests by tags, both are still being configured to support all requirements.

Reply:
Solved